.github/workflows/docker-build-and-push.yml

```
@@ -0,0 +1,415 @@
+        description: "Docker build context path"
+        type: string
+        default: "."
+      dockerfile:
+        description: "Path to Dockerfile"
+        type: string
+        default: "Dockerfile"
+      platforms:
+        description: "Platforms to build for (comma-separated)"
+        type: string
+        default: "linux/amd64, linux/arm64"
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)